### PR TITLE
API terminology and formatting updates for the 4.5 Release Notes

### PR DESCRIPTION
**Pull request by jeana-redhat**
API terminology and formatting updates for the 4.5 Release Notes
Preview: https://terms-form-update-rns-4_5--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-5-release-notes.html
For the most part, I did not replace hard wraps with soft wraps! As Jason pointed out, it makes reviewing the rest of it extremely painful (sorry Alex for your 4.4 RN experience). If we want to un-wrap, I'll do it in a different PR later :slightly_smiling_face:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@openshift/team-documentation for review

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -1269,53 +1279,53 @@ default interface for the operating system is selected based on common template
...
 
-* Previously,adding arbitrary taints to nodes could cause problems related to the DNS operator's operand. Now the DNS operator's operand tolerates any taint added to a node. The operand runs on and updates /etc/hosts on all Linux node hosts. *Missing CNI default network* events may be observed when the operand starts on a node that is still initializing, but such errors are transient and can be ignored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1813479[*BZ#1813479*])
+* Previously, adding arbitrary taints to nodes could cause problems related to the DNS operator's operand. Now the DNS operator's operand tolerates any taint added to a node. The operand runs on and updates `/etc/hosts` on all Linux node hosts. *Missing CNI default network* events may be observed when the operand starts on a node that is still initializing, but such errors are transient and can be ignored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1813479[*BZ#1813479*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-5-release-notes.adoc:1310]: Operator should still be capitalized

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -1343,17 +1353,17 @@ When a Task Param of type `array` was encountered, it would cast the array to a
...
+* The `keepYoungerThan` parameter in an image pruning custom resource uses nanoseconds and cannot be configured to use a larger period of time. Nanoseconds are not an appropriate period to use in an image pruner. A new parameter has been added to the image pruning custom resource, `keepYoungerThanDuration` that replaces and overrides the `keepYoungerThan` parameter. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1835004[*BZ#1835004*])
 
-* The Image Registry Operator did not properly clean up the storage status when the user changes the operator to `Removed` state. As a result, when the user changed the operator back to `Managed`, the operator could not create a new storage pod. The operator was changed to properly clean up the storage status and the operator can create a new storage pod.
+* The Image Registry Operator did not properly clean up the storage status when the user changed the operator to `Removed` state. As a result, when the user changed the operator back to `Managed`, the operator could not create a new storage pod. The operator was changed to properly clean up the storage status and the operator can create a new storage pod.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-5-release-notes.adoc:1360]: Operator

**Comment by ahardin-rh** [comment]
LGTM!

File: release_notes/ocp-4-5-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-5-release-notes.adoc:1310]: thanks! found several more throughout :+1:

**Pull request merged**